Source/ProjectFirma.Web/Controllers/ProjectController.cs

```diff
@@ -54,6 +54,7 @@
 using DetailViewData = ProjectFirma.Web.Views.Project.DetailViewData;
 using Index = ProjectFirma.Web.Views.Project.Index;
 using IndexViewData = ProjectFirma.Web.Views.Project.IndexViewData;
+using ProjectFirma.Web.ReportTemplates;

```

Comment:
This using statement can be removed. I think this stuck around after we moved the generate report action to the report controller